kickstarter/android-oss
# [🚮] KSR10

# What ❓
KSR10 is over. This PR gets rid of the unused code.

# How to QA? 🤔
Run the app; there should be no easter eggs even if you change the date of the device.

# Story 📖
[Trello Story](https://trello.com/c/Nzrqwtue/1380-remove-ksr10-code)
